### PR TITLE
FIP-26 + 27

### PR DESCRIPTION
**Pull request by Joeysantoro**

### CODE REVIEW

File: proposals/dao/fip_26.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { expect } from "chai";
+
+const e18 = '000000000000000000';
```
[the remark this answers is not in the document]
Comment (Joeysantoro) [reply proposals/dao/fip_26.ts:3]: permanent moratorium on string e18. Long live wei constants e18

File: proposals/dao/fip_26.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const FEI_PER_USD = '9950';
+const BUFFER = '50';
+const DAI_CAP = `150000000${e18}`;
```
[the remark this answers is not in the document]
Comment (Joeysantoro) [reply proposals/dao/fip_26.ts:7]: excellent call

**Comment by sslivkoff** [comment]
lgtm :heavy_check_mark:

**Review by sslivkoff — APPROVED** [review]
:rocket: :rocket:

**Review by ditchfieldcaleb — APPROVED** [review]

**Pull request merged**